Fuga.php

```diff
@@ -8,10 +8,11 @@
 class Fuga {
 
     public function index() {
-        return true;
+        $i = 0;
```

Comment:
__[phpmd]__
* Avoid variables with short names like $i. Configured minimum length is 3.